packages/dev-test/utils.js

```diff
@@ -2,7 +2,7 @@ import {ec as EC} from "elliptic"
 import * as fcl from "@onflow/fcl"
 import {config, sansPrefix, send, tx, withPrefix} from "@onflow/fcl"
 import {SHA3} from "sha3"
-import {mapValuesToCode} from "../src"
+import {mapValuesToCode} from "@onflow/flow-cadut"
```

Comment:
Probably too hard to switch this name and probably out of the scope of this PR, but I feel like there could be something a little more descriptive than `toCode`?